signals/sender/sender_new_domain_L90D.yml

```diff
@@ -0,0 +1,4 @@
+name: "Sender: Domain registered less than 90 days ago"
+type: "query"
+source: |
+  beta.whois(sender.email.domain).days_old < 90
```

Comment:
Why not just make the signal on `.days_old` instead?

Reply:
yes... that would work too :|